src/game_renderer/GameRenderer.cpp

```diff
@@ -186,9 +187,33 @@ void GameRenderer::RenderIt(GLFWwindow* window,
   game_state->SetItemsInView(
       GameRenderer::GetObjectsInView(vfplane, level->getTree()));
 
+  // large far for sexy looks
+  P->popMatrix();
+  P->pushMatrix();
+  P->perspective(45.0f, aspect, 0.01f, 1000.0f);
```

Comment:
I understand now, the sky doesn't get drawn if we keep the near plane at 150 for the entire draw cycle. It doesn't get culled, its just outside the far plane.